tests.md

```diff
@@ -194,35 +188,52 @@ The content of this file is somewhat similar to this:
+```bash
+cat /home/wptestrunner/wordpress/junit.xml
+```
+
+At this point we can generate the reports by sending them to WordPress.org, if necessary. Even if you haven't included the WordPress user (see below for how to create it), you can still run this file.
 
 ```bash
-cat /tmp/wordpress/junit.xml
+php report.php
 ```
 
 ### Cleaning up the environment for other tests
 
-Now that we have the tests working, all that remains is to delete all the files that have been created so that we can start over. To do this we will execute the following command:
+Having the tests working, all that remains is to delete all the files that have been created so that we can start over. To do this, execute the following command:
 
 ```bash
-cd /tmp/phpunit-test-runner/
 php cleanup.php
 ```
 
+### Automating the execution
+
+Once this first manual test has been done, please automate all these steps in a script, since it is required that each of these steps is executed sequentially for each test execution.
+
+This script should be run every time there is a change / commit in the WordPress master. Many hosting companies use a cron to run the script every few hours / days to make the appropriate checks, or when a change is made.
+
 ### Improving the configuration
 
-We must not forget that the aim of this tool is to verify that our environment and infrastructure is the optimal one for WordPress to work, so, following the example above, we could make several improvements such as installing the extension of bcmath, gd, libsodium, mcrypt, mod_xml and imagick or utilities such as ghostscript and imagemagick.
+Do not forget that the aim of this tool is to verify that the environment and infrastructure is the optimal one for WordPress to work, so, following the example, could make several improvements such as installing the extension of `bcmath`, `gd`, `libsodium`, `mcrypt`, `mod_xml` and `imagick` or utilities such as `ghostscript` and `imagemagick`.
 
 The goal? To be error free and have the green light for the perfect configuration.
 
+\[alert\]Some tests may be skipped or there may be tests with some risk. It is normal for errors to occur even with a properly configured environment.\[/alert\]
+
 ## How to report: Creating your bot for WordPress.org
 
-If you want your test [results to appear on the WordPress.org page](https://make.wordpress.org/hosting/test-results/), you can create a user for that.
+If you / your company want the test [results to appear on the WordPress.org page](https://make.wordpress.org/hosting/test-results/), create a user for that.
+
+The first thing to do is [create a user on WordPress.org](https://login.wordpress.org/register). If your company is called *ExampleHostingCompany, Inc*, for example, call your user something like *examplehostingcmpanybot*. Keep in mind that the associated email account should be checked frequently, as emails will arrive regarding the possible operation of the tests.
```

Comment:
I really like this section!

One SUPER MINOR note: I don't think it's vital to merge, but I'm guessing you mean "examplehostingcompanybot" (added an `o`).

Reply:
using everywhere "myhostingbot"